ocs_ci/deployment/deployment.py

```diff
@@ -3444,6 +3449,106 @@ def deploy(self):
         else:
             self.enable_managed_serviceaccount()
 
+    @retry(ACMObservabilityNotEnabled, tries=10, delay=30)
+    def check_observability_status(self):
+        """
+        Check observability status
+
+        raises ACMObservabilityNotEnabled: if the cmd returns False, ACM observability is not enabled
+
+        """
+
+        acm_observability_status = bool(
+            run_cmd(
```

Comment:
run_cmd is a deprecated form of exec_cmd, we need to change replace it here

Reply:
done